napari/_qt/qt_main_window.py

```diff
@@ -947,6 +971,149 @@ def _delete_qt_window(self):
         self._qt_window.close()
         del self._qt_window
 
+    def show_preferences(self):
+        """Show the preferences dialog."""
+        dlg = PreferencesDialog(parent=self._qt_window)
+        dlg.sig_resized.connect(self._update_preferences_dialog_size)
+
+        palette = self.qt_viewer.viewer.palette
+        dlg.setStyleSheet(template("", **palette))
+        dlg.resize(self._preferences_dialog_size)
+        dlg.exec_()
+
+    def _load_window_settings(self):
+        """
+        Load window layout settings from configuration.
+        """
+        window_size = SETTINGS.get("application", "window_size")
+        state = SETTINGS.get("application", "window_state")
+        preferences_dialog_size = SETTINGS.get(
+            "application", "preferences_size"
+        )
+
+        position = SETTINGS.get("application", "window_position")
+
+        # It's necessary to verify if the window/position value is valid with the current screen.
+        width = position[0]
+        height = position[1]
```

Comment:
```suggestion
        width, height = position
```

That will make sure position is also of length 2

Reply:
Fixed.